langserver/langserver.py

```diff
@@ -624,6 +645,25 @@ def handle(self):
         s = LangServer(conn)
         s.run()
 
+def get_pip_package(module_path):
+    best_pkg, best_level = None, -1
+    for pkg in get_installed_distributions():
```

Comment:
what are the performance characteristics of `get_installed_distributions`?

Reply:
It should be no slower than running `pip list` in the project root directory, which I think is fine, because it's called once per symbol found in a `textDocument/xdefinition` request (which typically finds at most 1 symbol).